foss4g2022.html

```diff
@@ -250,6 +276,201 @@ <h2>All-in-one software suite</h2>
         <img src="img/hexagons_3d_white_outlier.png" class="stretch">
 </section>
 
+
+<section>
+    <h2>Integration with Jupyter Notebooks</h2>
+    <ul>
+        <li>Notebooks: Mix of text, code, results, images, &hellip;</li>
+        <li>Usage: Prototyping, reproducibility, tutorials, &hellip;</li>
+        <li>Now in GRASS GIS: Python functions for easy integration</li>
+    </ul>
+    <img src="img/jupyter_text_images_3d.png" class="stretch">
+    <p>by Caitlin Haedrich</p>
+</section>
+
+
+<section>
+    <h2>Integration with Jupyter Notebooks</h2>
+    <p>
+        Web map with base maps, zooming, customizations, &hellip;
+    </p>
+    <img src="img/jupyter_interactive_viewshed.png" class="stretch">
+    <p>
+        <a href="https://tinyurl.com/grass-rc2">tinyurl.com/grass-rc2</a> (run in Binder)
+    </p>
+    <p>by Vaclav Petras and Anna Petrasova</p>
+</section>
+
+
+<section>
+    <h2>Integration with Jupyter Notebooks</h2>
+    <p>
+        Time-series visualization
+    </p>
+    <img src="img/jupyter_timeseries_viewshed.gif" class="stretch">
+    <p>by Vaclav Petras and Anna Petrasova</p>
+</section>
+
+
+<section>
+    <h2>Integration with Jupyter Notebooks</h2>
+    <p>
+        3D visualization
+    </p>
+    <img src="img/jupyter_3d_viewshed.png" class="stretch">
+    <p>by Vaclav Petras and Anna Petrasova</p>
+</section>
+
+
+<section>
+    <h2>Semantic Labels for Rasters</h2>
+    <ul>
+        <li>
+            attach a labels to raster maps
+            <small>(e.g., <code>S2_1</code> or <code>red</code>)</small>
+            <ul>
+                <li>image classification can use sematic labels to identify bands</li>
+                <li>different scenes can then be classified if semantic labels are the same</li>
+            </ul>
+        </li>
+        <li>
+            organize bands within one spatio-temporal dataset
+        </li>
+    </ul>
+    <br>
+    <img src="img/i_superpixels_slic_ndvi.png" class="stretch">
```

Comment:
replace this image with eg https://github.com/neteler/grass-gis-talks-markus/blob/main/img/band_references_scheme.png

Reply:
yes, 64dce78f2e15f32015b9c3f081d3583e4a1d6c86